### PR TITLE
Add optional 'title' property to Checklist component

### PR DESCRIPTION
**Pull request by marcioos**
screen shot 2017-02-23 at 14 46 19
```javascript
<Checklist.Main title='TITLE'>
  <Checklist.Item>Item1</Checklist.Item>
  <Checklist.Item>Item2</Checklist.Item>
  <Checklist.Item>Item3</Checklist.Item>
</Checklist.Main>
```

### CODE REVIEW

**Comment by Nevon** [comment]
It feels like this should be a child component, not a prop. Something like `Checklist.Title`.

File: Checklist/index.jsx [hunk 1]
```diff
@@ -29,10 +31,21 @@ function ChecklistMain ({ chromeless, className, children, customize, style, sty
...
+    {title && renderTitle(title, classNames(classes.title))}
     {children}
   </ul>
 }
 
+function renderTitle (title, className) {
+  return (
+    <li>
+      <Paragraph.Primary className={className}>
+        <strong>{title}</strong>
```
Comment (Nevon) [inline Checklist/index.jsx]: Get rid of the `strong` and apply the style change on the class instead.

Comment (xaviervia) [inline Checklist/index.jsx]: We are rejecting this one now since the `Block.Installments` component already behaves like it was intended.

**Comment by marcioos** [comment]
@Nevon As @xaviervia mentioned, we can achieve the same behavior using `Block.Installments` with a chromeless `Checklist` inside. So no need for this change.

**Pull request closed without merging**